.gitignore

```
@@ -0,0 +1 @@
+config.toml
```

```suggestion
config.toml
prometheus.yml
```
Config file of prometheus is not needed in the repo